features/external_tables.md

```diff
@@ -56,3 +57,90 @@ On many Postgres providers, you can also set up this connection in the other dir
 ## Other External Table Engines
 
 Hydra is working on providing connections to other data sources in the near future. If you have a request, please reach out to us via our support channel.
+
+## MySQL External Tables
+
+MySQL External Tables are implemented using [`mysql_fdw`](https://github.com/EnterpriseDB/mysql_fdw).
+To set up a MySQL External Table from `psql`, run the following SQL, replacing `...` with your server's information.
+
+```sql
+CREATE EXTENSION mysql_fdw;
+
+CREATE SERVER remote_mysql_server FOREIGN DATA WRAPPER mysql_fdw OPTIONS (host '...', port '...');
+
+CREATE USER MAPPING FOR CURRENT_USER SERVER remote_mysql_server OPTIONS (username '...', password '...');
+```
+
+Same as `postgres_fdw`, you can now create foreign tables from your remote MySQL database.
```

Comment:
```suggestion
You can now create foreign tables from your remote MySQL database.
```

Avoid cross-references or assuming people have read other sections of the docs (people interested in MySQL ET are likely to have not read about Postgres ET).